falco/templates/_helpers.tpl

```diff
@@ -277,6 +280,9 @@ be temporary and will stay here until we move this logic to the falcoctl tool.
       name: rulesfiles-install-dir
     - mountPath: /etc/falcoctl
       name: falcoctl-config-volume
+      {{- with .Values.mounts.volumeMounts }}
```

Comment:
These volumes are for the falco container. For your use case, we need to add volume mounts sections in `values.yaml` under `falcoctl.artifact.install` and `falcoctl.artifact.follow`.